src/cdk/stepper/stepper.ts

```diff
@@ -167,6 +169,10 @@ export class CdkStepper {
     this._groupId = nextId++;
   }
 
+  ngAfterContentInit() {
+    this._stepsArray = this._steps.toArray();
```

Comment:
Can't do this since `_steps` may update, e.g. when an `ngIf` changes from `true` to `false`